src/php/lib/Grpc/BaseStub.php

```diff
@@ -75,6 +75,48 @@ public function getTarget() {
   }
 
   /**
+   * @param $try_to_connect bool
+   * @return int The grpc connectivity state
+   */
+  public function getConnectivityState($try_to_connect = false) {
+    return $this->channel->getConnectivityState($try_to_connect);
+  }
+
+  /**
+   * @param $timeout in microseconds
+   * @return bool true if channel is ready
+   */
+  public function watchForReady($timeout) {
```

Comment:
You should document that this will throw an error if the channel state becomes `FATAL_ERROR`.